app/controllers/assessments_controller.rb

```diff
@@ -39,48 +34,41 @@ def remove_evidence_post
   class FormResponsesNotFound < StandardError; end
 
   def find_assessment(id = nil)
-    find(Assessment, id || params[:assessment_id])
+    Assessment.find(id || params[:assessment_id])
   end
 
   def find_evidence(id = nil)
-    find(Evidence, id || params[:evidence_id])
+    assessment = find_assessment
+    assessment.evidence.find(id || params[:evidence_id])
   end
 
   def find_evidence_for_assessment(assessment)
-    assessment.evidence_id_list.map { |evidence_id| find(Evidence, evidence_id) }.compact
-  end
-
-  def find(model_class, id)
-    form_responses = session[:form_responses][model_class.name][id] || raise(FormResponsesNotFound)
-    model_class.new(form_responses) if !form_responses.nil?
+    assessment.evidence
   end
 
   def save(form_responses)
-    ((session[:form_responses] ||= Hash.new)[form_responses.class.name] ||= Hash.new)[form_responses.id] = form_responses.serializable_hash
+    form_responses.save
   end
 
-  def delete_assessment(assessment)
-    assessment.evidence_id_list.each do |evidence_id|
-      delete(Evidence, evidence_id)
-    end
-    delete(Assessment, assessment.id)
-  end
+  #def delete_assessment(assessment)
+  # assessment.evidence_id_list.each do |evidence_id|
+  #    delete(Evidence, evidence_id)
+  #  end
+  #  delete(Assessment, assessment.id)
+  #end
 
-  def delete(model_class, id)
-    session[:form_responses][model_class.name].delete(id)
-  end
+  #def delete(model_class, id)
+  #  session[:form_responses][model_class.name].delete(id)
+  #end
 
-  def all_assessments
-    session[:form_responses][Assessment.name].each_value.map { |form_responses| Assessment.new(form_responses) }.compact
-  end
 
-  def reap_old_assessments
-    assessments_sorted_by_date = all_assessments.sort_by(&:date_created)
-    all_but_most_recent = assessments_sorted_by_date[0..-(Rails.configuration.number_of_assessments_to_keep + 1)]
-    if !all_but_most_recent.nil?
-      all_but_most_recent.each do |assessment|
-        delete_assessment(assessment)
-      end
-    end
-  end
+  #def reap_old_assessments
```

Comment:
Let's remove this code; but leave the call in (commented out) to remind us that's still a thing we should, one day, do.